### PR TITLE
add tasks docs

### PR DESCRIPTION
**Pull request by TusharMohapatra07**
This commit is linked to #83. I have updated the docs for tasks cli. @flossypurse @dfarr please review the addition and suggest changes if any.

### CODE REVIEW

File: docs/operate/cli-reference/commands/tasks.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+id: tasks
+title: CLI tasks command
+sidebar_label: Tasks
+description: Mock the behaviour of application node in the Resonate Server.
+tags:
+  - cli
+  - tasks
+---
+
+The `tasks` command enables you to interact with the tasks framework by simulating the behavior of an application node.
```
Comment (flossypurse) [inline docs/operate/cli-reference/commands/tasks.mdx:11]:
```suggestion
The `tasks` command enables you to interact with the Task Framework by simulating the behavior of an Application Node.
```

File: docs/operate/cli-reference/commands/tasks.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```shell
+resonate tasks <sub-command> <args> [flags]
+```
+
+## complete
+
+Complete a task.
```
Comment (flossypurse) [inline docs/operate/cli-reference/commands/tasks.mdx:19]:
```suggestion
Complete a Task.
```

File: docs/operate/cli-reference/commands/tasks.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Flags:
+
+- [--counter](/operate/cli-reference/flags#--counter)
+- [--process-id](/operate/cli-reference/flags#--process-id)
+- [--ttl](/operate/cli-reference/flags#--ttl)
+
+## claim
+
+Claim one or more tasks associated with the given ID.
```
Comment (flossypurse) [inline docs/operate/cli-reference/commands/tasks.mdx:33]:
```suggestion
Claim one or more Tasks associated with the given ID.
```

File: docs/operate/cli-reference/commands/tasks.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+resonate tasks claim <id> [flags]
+```
+
+Flags:
+
+- [--counter](/operate/cli-reference/flags#--counter)
+
+## heartbeat
+
+Send heartbeat signals for tasks associated with the given process ID.
```
Comment (flossypurse) [inline docs/operate/cli-reference/commands/tasks.mdx:45]:
```suggestion
Send Heartbeat signals for Tasks associated with the given process ID.
```

File: docs/operate/cli-reference/flags.mdx [hunk 5]
```diff
@@ -518,3 +518,27 @@ Enables ignore-asserts mode.
 Used in:
 
 - [serve](/operate/cli-reference/commands/serve)
+
+### --counter
+
+The counter is a feature that keeps track of the number of tasks. It increments each time a new task is added and decrements when a task is completed or removed.
```
Comment (flossypurse) [inline docs/operate/cli-reference/flags.mdx:524]:
```suggestion
The counter is a feature that keeps track of the number of Tasks.
It increments each time a new Task is added and decrements when a Task is completed or removed.
```

File: docs/operate/cli-reference/commands/tasks.mdx [hunk 6]
```diff
@@ -0,0 +1,53 @@
+---
+id: tasks
+title: CLI tasks command
+sidebar_label: Tasks
+description: Mock the behaviour of application node in the Resonate Server.
```
Comment (flossypurse) [inline docs/operate/cli-reference/commands/tasks.mdx:5]:
```suggestion
description: Interact directly with the Task Framework to mock the behaviour of Application Node.
```

**Review by flossypurse — APPROVED** [review]
Looking great. TY!

**Comment by TusharMohapatra07** [comment]
@flossypurse done.

**Pull request merged**